models/certificate.js

```diff
@@ -1,4 +1,31 @@
-module.exports = (sequelize, { STRING, BOOLEAN, INTEGER, TEXT, ARRAY, DECIMAL, DOUBLE }) => {
```

Comment:
I think Certificate is accurate because it is actually storing the certificate claims from the blockchain.  We have a Channel table for the Channels that are created on spee.ch.  It parallels  how we have a Claim table for all the stream claims in the blockchain (i guess this table should be Streams to be fully parallel) and a File table for all the files held locally by spee.ch.

Reply:
@billbitt got it